Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Parent Dockerfile https://github.com/docker-library/mongo/blob/982328582c74dd2f0a9c8c77b84006f291f974c3/3.0/Dockerfile
+FROM mongo:3.4
+
+# Modify child mongo to use /data/db2 as dbpath (because /data/db wont persist the build)
+RUN mkdir -p /data/db2 \
+    && echo "dbpath = /data/db2" > /etc/mongodb.conf \
+    && chown -R mongodb:mongodb /data/db2
+
+COPY data/ /pharmacy-data/
+
+RUN mongod --fork --logpath /var/log/mongodb.log --dbpath /data/db2 --smallfiles \
```

Comment:
Can we add logpath to the config and use the config file whenever we use mongood (e.g. `mongod --config /etc/mongodb.conf --shutdown`) so we only specify the hardcoded dbpath/logpath when we create the config to avoid duplication.